element_calcium_imaging/imaging.py

```diff
@@ -85,20 +100,46 @@ class ProcessingMethod(dj.Lookup):
 
 @schema
 class ProcessingParamSet(dj.Lookup):
-    definition = """  #  Parameter set used for processing of calcium imaging data
-    paramset_idx:  smallint
+    """Parameter set used for the processing of the calcium imaging scans,
+    including both the analysis suite and its respective input parameters.
+
+    A hash of the parameters of the analysis suite is also stored in order
+    to avoid duplicated entries.
```

Comment:
Could this be modified to make the first line sub-88 char and include additional info in the following pgf?